public/app/features/dashboard/api/publicDashboardApi.ts

```diff
@@ -137,9 +140,13 @@ export const publicDashboardApi = createApi({
       }),
       providesTags: (result, _, email) => [{ type: 'ActiveUserDashboards', id: email }],
     }),
-    listPublicDashboards: builder.query<ListPublicDashboardResponse[], void>({
-      query: () => ({
-        url: '/dashboards/public-dashboards',
+    listPublicDashboards: builder.query<PublicDashboardListWithPagination, number | void>({
+      query: (page = 1) => ({
+        url: `/dashboards/public-dashboards?page=${page}&perpage=8`,
```

Comment:
I recall we talk about how many pubdashes should we show per page, I'm not sure if 8 is the best approach. In a big monitor, there is a lot of empty space. Do we have any advice from UX about how to tackle this decision?

Reply:
I didn't ask them. I can try it. I think the best way would be based on the screen but what about a tv for example? We would show too many items and we don't want that, since it's impossible to search